Makefile.am

```diff
@@ -7,6 +7,7 @@ EXTRA_DIST = pi-blaster.service
 sbin_PROGRAMS = pi-blaster
 pi_blaster_SOURCES = pi-blaster.c mailbox.c
 
+if !DEBIAN_PKG
```

Comment:
Can you also change the the init_d_SCRIPTS (you moved the file to the debian subfolder).